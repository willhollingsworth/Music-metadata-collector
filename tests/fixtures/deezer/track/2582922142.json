{
  "id": 2582922142,
  "readable": true,
  "title": "Kaili",
  "title_short": "Kaili",
  "title_version": "",
  "isrc": "DED621000416",
  "link": "https://www.deezer.com/track/2582922142",
  "share": "https://www.deezer.com/track/2582922142?utm_source=deezer&utm_content=track-2582922142&utm_term=0_1751371608&utm_medium=web",
  "duration": 282,
  "track_position": 3,
  "disk_number": 1,
  "rank": 213304,
  "release_date": "2010-04-16",
  "explicit_lyrics": false,
  "explicit_content_lyrics": 0,
  "explicit_content_cover": 2,
  "preview": "https://cdnt-preview.dzcdn.net/api/1/1/b/f/6/0/bf6ab09cf1f44d6c8a6685afeeb63bf6.mp3?hdnea=exp=1751372508~acl=/api/1/1/b/f/6/0/bf6ab09cf1f44d6c8a6685afeeb63bf6.mp3*~data=user_id=0,application_id=42~hmac=48b60b78b0a1f0f87daa316ed16640636938471deaa5d72ed582e52ea8b08eef",
  "bpm": 130.01,
  "gain": -8.6,
  "available_countries": [
    "AE",
    "AF",
    "AL",
    "AM",
    "AO",
    "AS",
    "AT",
    "AU",
    "AZ",
    "BA",
    "BD",
    "BE",
    "BF",
    "BG",
    "BH",
    "BI",
    "BJ",
    "BN",
    "BT",
    "BW",
    "BY",
    "CC",
    "CD",
    "CF",
    "CG",
    "CH",
    "CI",
    "CK",
    "CM",
    "CN",
    "CU",
    "CV",
    "CX",
    "CY",
    "CZ",
    "DE",
    "DJ",
    "DK",
    "DZ",
    "EE",
    "EG",
    "ER",
    "ES",
    "ET",
    "FI",
    "FJ",
    "FK",
    "FM",
    "FR",
    "GA",
    "GB",
    "GE",
    "GH",
    "GM",
    "GN",
    "GQ",
    "GR",
    "GS",
    "GU",
    "GW",
    "HK",
    "HR",
    "HU",
    "ID",
    "IE",
    "IL",
    "IN",
    "IO",
    "IQ",
    "IR",
    "IS",
    "IT",
    "JO",
    "JP",
    "KE",
    "KG",
    "KH",
    "KI",
    "KM",
    "KP",
    "KR",
    "KW",
    "KZ",
    "LA",
    "LB",
    "LK",
    "LR",
    "LS",
    "LT",
    "LU",
    "LV",
    "LY",
    "MA",
    "MD",
    "ME",
    "MG",
    "MH",
    "MK",
    "ML",
    "MM",
    "MN",
    "MP",
    "MR",
    "MS",
    "MT",
    "MU",
    "MV",
    "MW",
    "MY",
    "MZ",
    "NA",
    "NE",
    "NF",
    "NG",
    "NL",
    "NO",
    "NP",
    "NR",
    "NU",
    "NZ",
    "OM",
    "PG",
    "PH",
    "PK",
    "PL",
    "PN",
    "PS",
    "PT",
    "PW",
    "QA",
    "RO",
    "RS",
    "RU",
    "RW",
    "SA",
    "SB",
    "SC",
    "SD",
    "SE",
    "SG",
    "SI",
    "SJ",
    "SK",
    "SL",
    "SN",
    "SO",
    "ST",
    "SY",
    "SZ",
    "TC",
    "TD",
    "TG",
    "TH",
    "TJ",
    "TK",
    "TL",
    "TM",
    "TN",
    "TO",
    "TR",
    "TV",
    "TW",
    "TZ",
    "UA",
    "UG",
    "UM",
    "UZ",
    "VG",
    "VI",
    "VN",
    "VU",
    "WS",
    "YE",
    "ZA",
    "ZM",
    "ZW"
  ],
  "contributors": [
    {
      "id": 3892,
      "name": "Caribou",
      "link": "https://www.deezer.com/artist/3892",
      "share": "https://www.deezer.com/artist/3892?utm_source=deezer&utm_content=artist-3892&utm_term=0_1751371608&utm_medium=web",
      "picture": "https://api.deezer.com/artist/3892/image",
      "picture_small": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/56x56-000000-80-0-0.jpg",
      "picture_medium": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/250x250-000000-80-0-0.jpg",
      "picture_big": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/500x500-000000-80-0-0.jpg",
      "picture_xl": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/1000x1000-000000-80-0-0.jpg",
      "radio": true,
      "tracklist": "https://api.deezer.com/artist/3892/top?limit=50",
      "type": "artist",
      "role": "Main"
    }
  ],
  "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
  "track_token": "AAAAAWhjz1hoZOiY0Hkd1GP6CXjigXaAIWB0aAy6bfcLUorQ8ubDAmToEuMc-bUCrYlhZ2ak-EhwipW9kSZnf67ww0viRfXIh8dzuYroqdOut7D0G7n4TgQRfEwynRL82CpytmXm7zReVTzxV_xgtp8bMFkDj0Z4kozlO905Dd2ncOrpmt3g0IcvTt9525CrXYewdbx34LzvYG5zu1GAu4S6JPo44TanjfJUQke6YPmg9lyh9PTNm12AjXIkcQKxaHX2oltEbQkZAcxZE-HjdNqkj6NmyQiFzsvzpEhuhWtMmPmIUgpS-UU_d68kCo4pAFpAhvCuJKjdO2WvMHdPykwIvqJQizzKW95JCIj8ecpznp-FF12Ep3cEEInCtFso1VhnpQ3JXwoe2FJKiA",
  "artist": {
    "id": 3892,
    "name": "Caribou",
    "link": "https://www.deezer.com/artist/3892",
    "share": "https://www.deezer.com/artist/3892?utm_source=deezer&utm_content=artist-3892&utm_term=0_1751371608&utm_medium=web",
    "picture": "https://api.deezer.com/artist/3892/image",
    "picture_small": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/56x56-000000-80-0-0.jpg",
    "picture_medium": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/250x250-000000-80-0-0.jpg",
    "picture_big": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/500x500-000000-80-0-0.jpg",
    "picture_xl": "https://cdn-images.dzcdn.net/images/artist/373b284334c5d7514cd3994f4da19911/1000x1000-000000-80-0-0.jpg",
    "radio": true,
    "tracklist": "https://api.deezer.com/artist/3892/top?limit=50",
    "type": "artist"
  },
  "album": {
    "id": 523121232,
    "title": "Swim",
    "link": "https://www.deezer.com/album/523121232",
    "cover": "https://api.deezer.com/album/523121232/image",
    "cover_small": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/56x56-000000-80-0-0.jpg",
    "cover_medium": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/250x250-000000-80-0-0.jpg",
    "cover_big": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/500x500-000000-80-0-0.jpg",
    "cover_xl": "https://cdn-images.dzcdn.net/images/cover/c7e0d67b3e09a84f80e6bd45d336a4a4/1000x1000-000000-80-0-0.jpg",
    "md5_image": "c7e0d67b3e09a84f80e6bd45d336a4a4",
    "release_date": "2010-01-01",
    "tracklist": "https://api.deezer.com/album/523121232/tracks",
    "type": "album"
  },
  "type": "track"
}