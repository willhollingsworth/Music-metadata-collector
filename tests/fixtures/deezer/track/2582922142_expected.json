{
  "track_name": "Kaili",
  "track_id": 2582922142,
  "artist_name": "Caribou",
  "artist_id": 3892,
  "album_name": "Swim",
  "album_id": 523121232
}